skops/io/_general.py

```diff
@@ -232,6 +233,28 @@ def object_get_instance(state, src):
     return instance
 
 
+def method_get_state(obj: Any, save_state: SaveState):
+    # This method is used to persist methods bound to a different instance
```

Comment:
I know I insisted on the distinction of methods of _other_ instances but I now see that what I said is a bit misleading. A better distinction would be: methods that are stored as instance attributes. This can refer to methods on other objects (which is probably the most common case) but also methods stored on the same object (as with zipf). Could you please amend the comment to reflect that? Sorry for the inconvenience I caused.

Reply:
No need to apologise! I think you've hit on an important distinction. Let me know if the new wording is clearer :)